packages/contentstack-export/src/utils/logger.ts

```diff
@@ -134,7 +134,7 @@ function init(_logPath: string) {
 }
 
 export const log = async (config: ExportConfig, message: any, type: string) => {
-  const logsPath = config.data || __dirname;
+  const logsPath = config.data;
```

Comment:
the data will always be present as we are passing it every time we are creating the logs in case of error is thrown and if not then we have it for the path
`log({ data: exportDir } as ExportConfig, `The log has been stored at ${exportDir}`, 'info');`